java-checks/src/main/java/org/sonar/java/checks/EnumMutableFieldCheck.java

```diff
@@ -65,6 +64,25 @@ public void visitNode(Tree tree) {
       }
     }
   }
+  
+  private static boolean isNotStaticOrFinal(VariableTree variableTree, ModifiersTree modifiersTree) {
```

Comment:
Not detecting my last push?

commit 4e10991a424e0c1f20fd2d574ad5af51fd80ef36
Author: Fernando Garcia <fergarrui@gmail.com>
Date:   Tue Dec 6 20:33:01 2016 +0000

    SONARJAVA-1604 - fix sonar issue

commit 2f4b61ce0b82358a4c5d79ae35302355d71b5394
Author: Fernando Garcia <fergarrui@gmail.com>
Date:   Tue Dec 6 19:03:13 2016 +0000

    SONARJAVA-1604 - Requested changes